apps/scan/backend/src/store.ts

```diff
@@ -1063,4 +1063,37 @@ export class Store {
   ): DiagnosticRecord | undefined {
     return getMostRecentDiagnosticRecord(this.client, type);
   }
+
+  getElectricalTestingStatusMessages(): Array<{
+    component: string;
+    statusMessage: string;
+    updatedAt: string;
+  }> {
+    return this.client.all(
+      `
+      select
+        component,
+        status_message as statusMessage,
+        updated_at as updatedAt
+      from electrical_testing_status_messages
+      order by component asc
+      `
+    ) as Array<{ component: string; statusMessage: string; updatedAt: string }>;
+  }
+
+  setElectricalTestingStatusMessage(
+    component: string,
```

Comment:
Since this is the primary key, maybe worth making the component type an enum?

Reply:
Makes sense to me! Adjusted: https://github.com/votingworks/vxsuite/pull/6042/commits/22324acbfd0249145c3111d0826d1eff22a4002d